sort.php

```diff
@@ -61,4 +61,13 @@ function bubbleSort($sort_array, $checkEquality)
     $fileName = 'php://stdin';
 }
 
-echo implode("", bubbleSort(checkForNextLine(file($fileName)), $funcCallback));
\ No newline at end of file
+if (array_search("-o", $argv)) {
+    if (! empty($options['o'])) {
+        file_put_contents($options['o'], bubbleSort(checkForNextLine(file($fileName)), $funcCallback));
+    } else {
+        printf("Sort: This option required an Argument %s got instead!\n", "'null'");
+        exit(1);
+    }
+} else {
+    echo implode("", bubbleSort(checkForNextLine(file($fileName)), $funcCallback));
```

Comment:
You're calling bubbleSort(... more than once. Please avoid that by using variables.